src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -1308,6 +1309,54 @@ - (void)_invokeCommandWithEndpointID:(NSNumber *)endpointID
     std::move(*bridge).DispatchAction(self);
 }
 
+- (void)_invokeKnownCommandWithEndpointID:(NSNumber *)endpointID
+                                clusterID:(NSNumber *)clusterID
+                                commandID:(NSNumber *)commandID
+                           commandPayload:(id)commandPayload
+                       timedInvokeTimeout:(NSNumber * _Nullable)timeout
+              serverSideProcessingTimeout:(NSNumber * _Nullable)serverSideProcessingTimeout
+                            responseClass:(Class _Nullable)responseClass
+                                    queue:(dispatch_queue_t)queue
+                               completion:(void (^)(id _Nullable response, NSError * _Nullable error))completion
+{
+    if (![commandPayload respondsToSelector:@selector(_encodeAsDataValue:)]) {
```

Comment:
Honestly, I only added that part because it was not compiling when I just did the call blind, but adding this showed that the real issue was not including the right headers....  So we could just take this bit out too.